includes/class-wpcom-legacy-redirector.php

```diff
@@ -306,9 +303,9 @@ public static function check_if_404( $url ) {
 			$response = wp_remote_get( $url );
 			// If it was an error, try again with no SSL verification, in case it was a self-signed certificate: https://github.com/Automattic/WPCOM-Legacy-Redirector/issues/64.
 			if ( is_wp_error( $response ) ) {
-				$args     = [
+				$args     = array(
 					'sslverify' => false,
-				];
+				);
 				$response = wp_remote_get( $url, $args );
 			}
 		}
```

Comment:
The file below is a CapabilityTest.php - should that be in this PR?

Reply:
I am missing the file indication here, are you referring to the UtilsTest class?